### ISSUE TITLE
Documentation of Math library

### ISSUE THREAD

**Issue by JeanCollas** [dotnet/dotnet-api-docs#4951]
Every time I use the Sin/Cosin functions (with GPS coordinates) I wonder what should be the unit the input.
Even if it seems logical, in some contexts it is not, therefore it would be nice to have "in radians" in the IntelliSense documentation.
I couldn't find where the comments are located in the repository (or in another).

**Comment by tannergooding** [comment]
Thanks @carlossanlop! Just to confirm, it sounds like just updating the regular docs will also flow back to IntelliSense (if so, awesome!)?

**Comment by carlossanlop** [comment]
Correct. Make the change in dotnet-api-docs and we will take care of updating the dotnet/runtime intellisense on the next release.
